content/projects/gsoc/2020/project-ideas/jenkins-distribution-customize-service.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: gsocprojectidea
+title: "Jenkins distribution customize service"
+goal: "Provide an out of the box solution for Jenkins users"
```

Comment:
```suggestion
goal: "Provide an out of the box solution for packaging Jenkins distributions as WAR files or Docker images"
```

Reply:
If you consider other output types, I suggest adding them explicitly

Reply:
Good point. Consider the scope of GSoC, I think that two types are enough.